graphql/src/runner.rs

```diff
@@ -135,12 +146,30 @@ where
     ) -> Self {
         let logger = logger.new(o!("component" => "GraphQlRunner"));
         let result_size = Arc::new(ResultSizeMetrics::new(registry));
+        let mut graphql_validation_plan = ValidationPlan { rules: Vec::new() };
+
+        if !(*DISABLE_GRAPHQL_VALIDATIONS) {
+            graphql_validation_plan.add_rule(Box::new(UniqueOperationNames {}));
+            graphql_validation_plan.add_rule(Box::new(LoneAnonymousOperation {}));
+            graphql_validation_plan.add_rule(Box::new(SingleFieldSubscriptions {}));
+            graphql_validation_plan.add_rule(Box::new(KnownTypeNames {}));
+            graphql_validation_plan.add_rule(Box::new(FragmentsOnCompositeTypes {}));
+            graphql_validation_plan.add_rule(Box::new(VariablesAreInputTypes {}));
+            graphql_validation_plan.add_rule(Box::new(LeafFieldSelections {}));
+            graphql_validation_plan.add_rule(Box::new(FieldsOnCorrectType {}));
+            graphql_validation_plan.add_rule(Box::new(UniqueFragmentNames {}));
+            graphql_validation_plan.add_rule(Box::new(KnownFragmentNames {}));
+            graphql_validation_plan.add_rule(Box::new(NoUnusedFragments {}));
+            graphql_validation_plan.add_rule(Box::new(OverlappingFieldsCanBeMerged {}));
+        }
```

Comment:
To me, the validation plan is an implementation detail of how we do validation, and it might be better to just make that a `lazy_static` in `query.rs` itself. I don't think there's much value in having it as a field in the GraphQL runner.

Reply:
agree, 100000%.